packages/components/src/HtmlRenderer/Models.purs

```diff
@@ -108,31 +109,35 @@ modifyingHook = toHookRep <<< ModifyingHook
 instance modifyingHookToGenericHook :: ToGenericHook ModifyingHook where
   toGenericHook (ModifyingHook { shouldProcessNode, processNode }) =
     { replaceChildren: false
-    , shouldProcessNode
-    , processNode: Just processNode
-    , processNodeWithReplacement: Nothing
+    , shouldPreprocessNode: Just shouldProcessNode
+    , shouldProcessNode: Just shouldProcessNode
```

Comment:
Can this be `Nothing`?

Reply:
Yes indeed. A left over from my testing.